YcmdCompletion.sublime-settings

```diff
@@ -46,4 +46,19 @@
     and other tools, uncomment this:
   */
   // "python_binary_path": "/usr/local/bin/python"
+
+  /*
+    The languages you wish to use ycmd for.
+    Supported: cpp, rust
+
+    The following languages may also work, but are untested with this plugin:
+    c, erlang, java, d, lua, python,
+    ocaml, js, perl, go, objc,
+    cpp, cs, php, ruby, rust
+
+    These languages may also work if the corresponding Sublime
+    syntax plugin is installed:
+    vb, typescript, elixir
+  */
```

Comment:
I'm talking only about this case:
* User sets ```"languages": ["foo"]``` in settings
* Language ```foo``` is not supported (not found in ```LANG_MAP```), so in ```filetype``` param to ycmd ```foo``` is passed. 
* User gets non-working plugin without proper error message

So I propose to validate this foo to be in a supported list. You can extend list of supported languages to whatever set you like, but if user tries to use different value, he'll get an error with description.

Is it clear now?